backend/dataall/modules/datapipelines/cdk/datapipelines_cdk_pipeline.py

```diff
@@ -215,23 +228,17 @@ def git_push_repo(self):
     @staticmethod
     def clean_up_repo(path):
         if path:
-            precmd = [
-                'deactivate;',
-                'rm',
-                '-rf',
-                f"{path}"
-            ]
-
+            cmd = ['rm', '-rf', f"{path}"]
```

Comment:
I removed the `deactivate` command. I think we do not need it. I am testing if there are any implications when deploying to AWS

Reply:
Tested in AWS and working fine from my end